### PR TITLE
cdc: add some troubleshooting FAQs.

### PR DESCRIPTION
**Pull request by liuzix**
### What is changed, added or deleted? (Required)

- Added some faqs to aid trouble shooting TiCDC.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/troubleshoot-ticdc.md [hunk 1]
```diff
@@ -344,3 +344,55 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
     ```shell
     cdc cli changefeed resume -c test-cf --pd=http://10.0.10.25:2379
     ```
+
+## 从 CDC 中删除了一个 checkpoint 滞后的 changefeed 后，TiKV 的 `gc-safepoint` 出现大幅推进，导致 TiKV 性能下降，如何解决？
+
+CDC 会在 PD 中设置 `gc-safepoint` 保证可能将被同步的数据不被 TiKV 垃圾回收机制清除，该 `gc-safepoint` 以进度最落后的 changefeed 为准。如果一个 changefeed 落后了若干天，之后用户将该 changefeed 删除，`gc-safepoint` 会突然前进若干天，导致 TiKV 在短时间内 GC 大量数据，影响正常业务。这种情况下可以通过配置 `gc.max-write-bytes-per-sec` 限制 GC worker 每秒数据写入量，降低对正常请求的影响。可以通过 tikv-ctl 动态修改
```
Comment (amyangfei) [inline ticdc/troubleshoot-ticdc.md:350]: can add a link to /garbage-collection-configuration#gc-io-limit

File: ticdc/troubleshoot-ticdc.md [hunk 2]
```diff
@@ -344,3 +344,55 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+
+```
+tikv-ctl --host=ip:port modify-tikv-config -n gc.max_write_bytes_per_sec -v 10MB
+```
+
+上述命令中 ip:port 指的是 TiKV 节点的地址, 可能需要在所有节点上运行， 10MB 可以改为其他适当的数值。
+
+## TiCDC 多节点负载分配不均怎么解决？
+
+出于稳定性考虑，TiCDC 新建的 changefeed 中默认不开启定期自动再平衡。再平衡只在新节点加入时触发。此外 TiCDC 在调度表时会尽量保证轻负载节点优先。如果偶然出现了不平衡现象，可以尝试通过触发
```
Comment (amyangfei) [inline ticdc/troubleshoot-ticdc.md:360]: 是否也可以通过手动调度某张表来调节负载均衡

File: ticdc/troubleshoot-ticdc.md [hunk 3]
```diff
@@ -393,3 +451,9 @@ cdc cli changefeed update -c test-cf --pd=http://10.0.10.25:2379 --start-ts 4152
 
 cdc cli changefeed resume -c test-cf --pd=http://10.0.10.25:2379
 ```
+
+> **注意：**
+>
+> 以上步骤仅适用于 TiCDC v4.0.11 及以上版本（不包括 v5.0.0-rc）。
+> 在其它版本中（v4.0.11 以下和 v5.0.0-rc），DDL 执行失败后 changefeed 的 checkpoint-ts 为该 DDL 语句的 finish-ts。使用 `cdc cli changefeed resume` 恢复同步任务后不会重试该 DDL 语句，而是直接跳过执行该 DDL 语句。
+
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:459]:
```suggestion
```

File: ticdc/troubleshoot-ticdc.md [hunk 4]
```diff
@@ -345,6 +345,64 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
     cdc cli changefeed resume -c test-cf --pd=http://10.0.10.25:2379
     ```
 
+## 从 CDC 中删除了一个 checkpoint 滞后的 changefeed 后，TiKV 的 `gc-safepoint` 出现大幅推进，导致 TiKV 性能下降，如何解决？
```
Comment (overvenus) [inline ticdc/troubleshoot-ticdc.md:348]:
```suggestion
## 从 TiCDC 中删除了一个 checkpoint 滞后的 changefeed 后，TiKV 的 `gc-safepoint` 出现大幅推进，导致 TiKV 性能下降，如何解决？
```

File: ticdc/troubleshoot-ticdc.md [hunk 5]
```diff
@@ -345,6 +345,64 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+
+CDC 会在 PD 中设置 `gc-safepoint` 保证可能将被同步的数据不被 TiKV 垃圾回收机制清除，该 `gc-safepoint` 以进度最落后的 changefeed 为准。如果一个 changefeed 落后了若干天，之后用户将该 changefeed 删除，`gc-safepoint` 会突然前进若干天，导致 TiKV 在短时间内 GC 大量数据，影响正常业务。这种情况下可以通过配置 `gc.max-write-bytes-per-sec` 限制 GC worker 每秒数据写入量，降低对正常请求的影响。可以通过 tikv-ctl 动态修改
```
Comment (overvenus) [inline ticdc/troubleshoot-ticdc.md:350]:
```suggestion
TiCDC 会在 PD 中设置 `gc-safepoint` 保证可能将被同步的数据不被 TiKV 垃圾回收机制清除，该 `gc-safepoint` 以进度最落后的 changefeed 为准。如果一个 changefeed 落后了若干天，之后用户将该 changefeed 删除，`gc-safepoint` 会突然前进若干天，导致 TiKV 在短时间内 GC 大量数据，影响正常业务。这种情况下可以通过配置 `gc.max-write-bytes-per-sec` 限制 GC worker 每秒数据写入量，降低对正常请求的影响。可以通过 tikv-ctl 动态修改。
```

File: ticdc/troubleshoot-ticdc.md [hunk 6]
```diff
@@ -345,6 +345,64 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+
```
Comment (overvenus) [inline ticdc/troubleshoot-ticdc.md:351]:
```suggestion

{{< copyable "shell-regular" >}}

```

File: ticdc/troubleshoot-ticdc.md [hunk 7]
```diff
@@ -345,6 +345,64 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+```
+tikv-ctl --host=ip:port modify-tikv-config -n gc.max_write_bytes_per_sec -v 10MB
+```
+
+上述命令中 ip:port 指的是 TiKV 节点的地址, 可能需要在所有节点上运行， 10MB 可以改为其他适当的数值。更多信息请参阅 [垃圾回收配置](/garbage-collection-configuration#gc-io-limit)。
+
+## TiCDC 多节点负载分配不均怎么解决？
+
+出于稳定性考虑，TiCDC 新建的 changefeed 中默认不开启定期自动再平衡。再平衡只在新节点加入时触发。此外 TiCDC 在调度表时会尽量保证轻负载节点优先。如果偶然出现了不平衡现象，可以尝试通过触发 
+
```
Comment (overvenus) [inline ticdc/troubleshoot-ticdc.md:361]:
```suggestion

{{< copyable "shell-regular" >}}

```

File: ticdc/troubleshoot-ticdc.md [hunk 8]
```diff
@@ -345,6 +345,64 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+```
+curl -X POST -d 'cf-id=[your-changefeed-id]' http://owner:port/capture/owner/rebalance_trigger
+```
+
+来解决。
+
+此外，还可以通过手工调度的方式来进行平衡
+
```
Comment (overvenus) [inline ticdc/troubleshoot-ticdc.md:369]:
```suggestion

{{< copyable "shell-regular" >}}

```

File: ticdc/troubleshoot-ticdc.md [hunk 9]
```diff
@@ -345,6 +345,64 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
...
+请使用 `pd-ctl service-gc-safepoint --pd <pd-addrs>` 指令查询当前的 `gc_safe_point` 与 `service_gc_safe_points`. 如果 `gc_safe_point` 小于 TiCDC 的 changefeed 的 checkpoint，则直接在 create changefeed 指令后加上 `--disable-gc-check` 参数创建 changefeed 即可。
+
+> **注意：**
+> 如果上述条件不满足 (如果 PD 并非有 bug 的版本或 `gc_safe_point` 大于当前 checkpoint)，为了避免 TiCDC 遗漏数据的风险，请联系 PingCAP 或社区技术支持。
+
+## 使用 TiCDC 创建 changefeed 时，`enable-old-value` 设置为 true 后 cdc 执行到下游的语句仍然为 `REPLACE INTO` 而非 `UPDATE`, 这是预期的吗？
+
+创建 changefeed 时需要显式指定 `safe-mode` 为 `false` 才会让 MySQL sink 使用 `update`, 执行步骤为:
+
```
Comment (overvenus) [inline ticdc/troubleshoot-ticdc.md:394]:
```suggestion

{{< copyable "shell-regular" >}}

```

**Review by overvenus — COMMENTED** [review]
rest lgtm

**Comment by TomShawn** [comment]
To better manage PR in this repo, I'll close this one for now. Feel free to reopen it if necessary. Thanks~

**Pull request closed without merging**